### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by SiddBW09**
## What does this PR do?
Add resource(s) 

### Description
Added a Matlab Hindi Tutorial
### Why is this valuable (or not)?
It teaches hindi speakers how to code in matlab
### How do we know it's really free?
It is a youtube video
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by SiddBW09** [comment]
I don't know why the blank line test is failing.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -484,6 +484,7 @@
 
 * [Matlab Complete Course](https://www.youtube.com/watch?v=iS5J4TlLSEM) - Armughan Ali
 * [MATLAB Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfdIlwG649bdzVHM4iLbD_H) - WsCube Tech
+* [MATLAB Tutorial Series in Hindi](https://www.youtube.com/playlist?list=PL6G0RjixRx3xgzxqjRdZIoLLpzhyLFTXk) - WittyRobo
```
Comment (eshellman) [inline courses/free-courses-hi.md:487]:
```suggestion
* [MATLAB (HINDI) Tutorial Series](https://www.youtube.com/playlist?list=PL6G0RjixRx3xgzxqjRdZIoLLpzhyLFTXk) - WittyRobo
```
Don't invent titles! Use the title given by the resource.

**Comment by SiddBW09** [comment]
I don't know why the alphabet test is failing.

**Comment by SiddBW09** [comment]
Done, sorry for the inconvenience.

**Pull request merged**